spyder/api/plugin_registration/_confpage.py

```diff
@@ -62,10 +62,17 @@ def setup_page(self):
         self.external_plugins_group = QGroupBox(_("External plugins"))
 
         i = 0
+        # Temporal fix to avoid disabling external plugins.
+        # for more info see spyder#17464
+        can_be_disabled = False
```

Comment:
Maybe the variable could be named something like `show_external_plugins_group`?

Reply:
Sure!